internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,443 @@
+			}
+			output.Printf(errors.UpdatedResourceMsg, resource.Topic, topicName)
+			if description != "" {
+				if err = addTopicDescription(details.srClient, details.srContext, details.clusterId+":"+topicName,
+					description); err != nil {
+					return true, false, fmt.Errorf("unable to update topic description: %v", err)
+				}
+			}
+			return true, false, nil
+		}
+	}
+	log.CliLogger.Infof("Topic not found. Adding a new topic: %s", topicName)
+	if _, httpResp, err := kafkaRest.CloudClient.CreateKafkaTopic(details.clusterId,
+		createTopicRequestData); err != nil {
+		restErr, parseErr := kafkarest.ParseOpenAPIErrorCloud(err)
+		if parseErr == nil && restErr.Code == ccloudv2.BadRequestErrorCode {
+			// Print partition limit error w/ suggestion
+			if strings.Contains(restErr.Message, "partitions will exceed") {
+				return false, false, errors.NewErrorWithSuggestions(restErr.Message, errors.ExceedPartitionLimitSuggestions)
+			}
+		}
+		return false, false, kafkarest.NewError(kafkaRest.CloudClient.GetUrl(), err, httpResp)
+	}
+	output.Printf(errors.CreatedResourceMsg, resource.Topic, topicName)
+	return false, true, nil
+}
+
+func addTopicDescription(srClient *srsdk.APIClient, srContext context.Context, qualifiedName string, description string) error {
+	atlasEntity := srsdk.AtlasEntityWithExtInfo{
+		ReferredEntities: nil,
+		Entity: srsdk.AtlasEntity{
+			Attributes: map[string]any{
+				"description":   description,
+				"qualifiedName": qualifiedName,
+			},
+			TypeName: "kafka_topic",
+		},
+	}
+	_, err := srClient.DefaultApi.PartialUpdateByUniqueAttributes(srContext,
+		&srsdk.PartialUpdateByUniqueAttributesOpts{AtlasEntityWithExtInfo: optional.NewInterface(atlasEntity)})
+	return err
+}
+
+func resolveSchemaType(contentType string) string {
+	if strings.Contains(contentType, "avro") {
+		return "AVRO"
+	} else if strings.Contains(contentType, "json") {
+		return "JSON"
+	} else {
+		return "PROTOBUF"
+	}
+}
+
+func registerSchema(details *accountDetails, topicName string, components Components) (int32, error) {
+	// Registering the schema
+	// Subject name follows the TopicNameStrategy
+	subject := topicName + "-value"
+	if components.Messages != nil && components.Messages[strcase.ToCamel(topicName)+"Message"].Payload != nil {
+		schema, err := yaml.Marshal(components.Messages[strcase.ToCamel(topicName)+"Message"].Payload)
+		if err != nil {
+			return 0, err
+		}
+		jsonSchema, err := yaml3.ToJSON(schema)
+		if err != nil {
+			return 0, fmt.Errorf("failed to convert schema to JSON: %v", err)
+		}
+		id, _, err := details.srClient.DefaultApi.Register(details.srContext, subject,
+			srsdk.RegisterSchemaRequest{
+				Schema:     string(jsonSchema),
+				SchemaType: resolveSchemaType(components.Messages[strcase.ToCamel(topicName)+"Message"].ContentType),
+			})
+		if err != nil {
+			return 0, fmt.Errorf("unable to register schema: %v", err)
+		}
+		output.Printf("Schema registered under subject %s with ID %d.\n", subject, id.Id)
```

Comment:
Please put quotes around all resources!
```suggestion
		output.Printf("Schema registered under subject "%s" with ID "%d".\n", subject, id.Id)
```